datahub-web-react/src/app/settings/SettingsPage.tsx

```diff
@@ -128,6 +127,6 @@ export const SettingsPage = () => {
                     ))}
                 </Switch>
             </PageContainer>
-        </SearchablePage>
+        </>
```

Comment:
can also remove here (and a few others at the top of this PR)

Reply:
yep!